docsrc/content/Matrix_Vector.fsx

```diff
@@ -13,86 +14,140 @@ open FSharp.Plotly
 <a name="Vector"></a>
 
 ##Vector
+*)
+#r "FSharp.Stats.dll"
+open FSharp.Stats
 
-<a name="Matrix"></a>
+Array.set
 
-##Matrix
+let v = 
+    vector [|2.0; 20.0; 1.|]
 
+(*** include-value:v ***)
 
+let rv = 
+    rowvec [|2.0; 20.0; 1.|]
+
+(*** include-value:rv ***)
+
+(**
+<a name="Matrix"></a>
+
+##Matrix
 
 *)
-//#r "D:/Source/FSharp.Stats/bin/FSharp.Stats.dll"
 #r "FSharp.Stats.dll"
-//open FSharp
-open Microsoft.FSharp.Math
 open FSharp.Stats
-open FSharp.Stats.Algebra
-
 
 
 // http://fdatamining.blogspot.de/2010/03/matrix-and-linear-algebra-in-f-part-i-f.html
-// 1.9.7.8. 
 // http://fdatamining.blogspot.de/search/label/linear%20algebra
 
 let A = 
     matrix [ [ 1.0; 7.0; 2.0 ]
              [ 1.0; 3.0; 1.0 ]
              [ 2.0; 9.0; 1.0 ] ]
 
+(*** include-value:A ***)
+
 let B = 
     matrix [ [ 10.0; 70.0; 20.0 ]
              [ 10.0; 30.0; 10.0 ]
              [ 20.0; 90.0; 10.0 ] ]
 
-
-let v = 
-    vector [|2.0; 20.0; 1.|]
+(*** include-value:B ***)
 
 
-let rv = 
-    rowvec [|2.0; 20.0; 1.|]
-
+(***hide***)
+let exmp1 =
+    A
+    |> Matrix.Generic.enumerateRowWise (Seq.sum)
 
 A
 |> Matrix.Generic.enumerateRowWise (Seq.sum)
 
+(*** include-value:exmp1 ***)
+
+(***hide***)
+let exmp2 =
+    A
+    |> Matrix.mapiCols (fun i r -> r)
+
 A
 |> Matrix.mapiCols (fun i r -> r)
 
-//let f x = x**2.
+(*** include-value:exmp2 ***)
+
+(***hide***)
+let exmp3 = A + B
+let exmp4 = A - B
+let exmp5 = A * B
+let exmp6 = A.*B
+let exmp7 = A * 2.0
+let exmp8 = 2.0 * A
+let exmp9 = -A
+let exmp10 = A * v
+let exmp11 = Matrix.dot A B
+let exmp12 = A + 1.
+let exmp13 = v + 1.
+let exmp14 = rv + 1.
 
-//"2.;3.;".Split(';') 
-//|> Array.filter (fun x -> x <> "") 
-//|> Array.map float 
-//|> Array.map f
+A + B
 
+(*** include-value:exmp3 ***)
 
+A - B
+
+(*** include-value:exmp4 ***)
+
+A * B // matrix product
```

Comment:
Please consider commenting the functionalities even further, so that the documentation structure becomes clearer.